### ISSUE TITLE
promisify a function has More than two params

### ISSUE THREAD

**Issue by HoKangInfo** [suguru03/aigle#59]
`v1.11`

promisify a function that callback function has More than two params

e.q.
```
const { exec } = require('child_process');
const exec_1 =  require('util').promisify(exec);
const exec_2 = require('aigle').promisify(exec);

const cmd = 'ls 1>&2';
exec_1(cmd).then(raw=> console.log('exec 1=>',raw)).catch(e=>console.log('exec 1 error=>',e));
exec_2(cmd).then(raw=> console.log('exec 2=>',raw)).catch(e=>console.log('exec 2 error=>',e));
```

.result
```
exec 1 => { stdout: '',  stderr: 'a.file'}
exec 2 => 
```

**Comment by HoKangInfo** [comment]
Thank for your reply. 
I understand child_process.exec is non-standard callback. 
Just to discuss need or not support.

**Comment by suguru03** [comment]
@HoKangInfo @thammin Thanks for the issue, it is so helpful!
Yes, let's support it! 😄 

> we need to implement this in Node.js environment only and just skip the checks in browser.

Sure, we need to handle it. 👍

**Comment by suguru03** [comment]
I published it on `v1.12.0-alpha.7`.
But I just realized the promisified function doesn't return Aigle instances. I'll think about it more.

**Comment by suguru03** [comment]
I published it on `v1.12.0`!
